doc/deploy.md

```diff
@@ -120,3 +119,21 @@ status: SERVING
 grpc-health-probe -tls -addr grpc2.testnet.xmtp.network:443
 status: SERVING
 ```
+
+## Deploy a new XMTP testnet contract
```

Comment:
When will an xmtpd need to do this?

Reply:
Ideally, never. The local development environment should make use of a chain container image with all the contracts already deployed in deterministic addresses.

Reply:
@fbac  Based on your answer, I've deleted the "Deploy a new XMTP testnet contract" section from this doc. It sounds like including it might cause confusion? <3

Reply:
Perfect. :)